exasol_sphinx_github_pages_generator/deploy_github_pages.py

```diff
@@ -7,7 +7,7 @@
 from inspect import cleandoc
 
 
-def deploy_github_pages(argv):
+def deploy_github_pages(argv=sys.argv[1:]):
```

Comment:
:warning: Don't! This is a very dangerous construct! :warning: 

Never put a mutable object as default argument!
if you google for `python mutable defaults` you will find a lot of insights e.g.: 
   * [python mutable defaults the root of all evil](https://florimond.dev/en/posts/2018/08/python-mutable-defaults-are-the-source-of-all-evil/)